sdk/identity/azure-identity/tests/test_browser_credential.py

```diff
@@ -100,7 +101,7 @@ def test_disable_automatic_authentication():
     empty_cache = TokenCache()  # empty cache makes silent auth impossible
     transport = Mock(send=Mock(side_effect=Exception("no request should be sent")))
     credential = InteractiveBrowserCredential(
-        disable_automatic_authentication=True, transport=transport, _cache=empty_cache
+        _disable_automatic_authentication=True, transport=transport, _cache=empty_cache
     )
```

Comment:
Seems to me they are invalid scenarios. Do we want to keep them?

Reply:
The tests? I think we do. We're disconnecting the features from the public API, not removing them. We want them to continue functioning because we'll make them public again in the next beta.

Reply:
My concern is user uses this as reference or our samples.

But again, I will let you and @schaabs make the final call.